src/NuGet.Core/NuGet.Protocol/Resources/PackageSearchResourceV3.cs

```diff
@@ -108,6 +108,9 @@ private static IEnumerable<VersionInfo> GetVersions(PackageSearchMetadata metada
                     Common.ILogger log,
                     CancellationToken cancellationToken)
         {
+            //Console.Error.WriteLine($"Found {_searchEndpoints.Length} search endpoints.");
+            log.LogVerbose($"Found {_searchEndpoints.Length} search endpoints.");
```

Comment:
Does this add actual value or was it used for debugging?

Reply:
This, along with the "Querying <url>" line were already there in this Search method. I wanted the querying url output to print only for `-Verbosity detailed`, so I changed the way it was printed to use logger. I did the same for the "Found ... endpoints" line only because the previous version of this method was treating both the same way. Personally, I'm not sure if the user would find this line useful in the output, but I didn't want to get rid of it in case it was actually important.